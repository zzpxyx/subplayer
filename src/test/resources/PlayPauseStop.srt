1
00:00:00,100 --> 00:00:00,200
a

2
00:00:00,300 --> 00:00:00,400
b

