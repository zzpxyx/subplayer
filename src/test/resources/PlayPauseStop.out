100^a
400^
450^
650^a
750^
850^b
950^
